firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteIndexManager.java

```diff
@@ -66,6 +72,12 @@
   private final LocalSerializer serializer;
   private final String uid;
 
+  /**
+   * Maps from a target to its equivalent list of sub-targets. Each sub-target contains only one
+   * term from the target's disjunctive normal form (DNF).
+   */
+  private final Map<Target, List<Target>> targetToDnfSubTargets = new HashMap<>();
```

Comment:
We probably need to find a way to tie this list into our GC algorithm so that we remove the mapping once we remove the target. Probably worth a TODO.